app/Http/Controllers/FileController.php

```diff
@@ -673,7 +673,10 @@ function createFolder(Request $request){
 			Log::critical("Could not create folder. Please debug");
 		}
 
-		return ['message' => 'Ok', 'success' => $success];
+
+        return redirect()->route('file.show',$parent);
```

Comment:
if parent is null then it should be redirected to home. Ill fix this in a later commit,